ethereum/executionengine/src/main/java/tech/pegasys/teku/ethereum/executionengine/Web3jHttpClient.java

```diff
@@ -11,37 +11,41 @@
  * specific language governing permissions and limitations under the License.
  */
 
-package tech.pegasys.teku.ethereum.executionlayer.client;
+package tech.pegasys.teku.ethereum.executionengine;
 
 import java.net.URI;
+import java.time.Duration;
 import java.util.Optional;
 import okhttp3.OkHttpClient;
 import okhttp3.logging.HttpLoggingInterceptor;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.web3j.protocol.Web3jService;
 import org.web3j.protocol.http.HttpService;
-import tech.pegasys.teku.ethereum.executionlayer.client.auth.JwtAuthHttpInterceptor;
-import tech.pegasys.teku.ethereum.executionlayer.client.auth.JwtConfig;
+import tech.pegasys.teku.ethereum.executionengine.auth.JwtAuthHttpInterceptor;
+import tech.pegasys.teku.ethereum.executionengine.auth.JwtConfig;
 import tech.pegasys.teku.infrastructure.time.TimeProvider;
 
 public class Web3jHttpClient extends Web3JClient {
   private static final Logger LOG = LogManager.getLogger();
 
   public Web3jHttpClient(
-      final URI endpoint, final TimeProvider timeProvider, final Optional<JwtConfig> jwtConfig) {
+      final URI endpoint,
+      final TimeProvider timeProvider,
+      final Optional<Duration> timeout,
```

Comment:
I don't think timeout needs to be optional. In production code as far as I can tell we always set a timeout so it's probably worth just making it required - it's good to be explicit about what the timeout is rather than just depending on OkHttp defaults anyway.